### PR TITLE
Piyush_Assignment_21_30Min

### PR DESCRIPTION
**Pull request by PiyushPJun20**
image

image

### CODE REVIEW

File: src/Fibonacci.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class Fibonacci {
+
+	public static void main(String[] args) {
+		int n, first = 0, next = 1;
+		System.out.println(" Enter the value of n : ");
+		Scanner scanner = new Scanner(System.in);
+		n = scanner.nextInt();
+		System.out.print(" Fibonacci Series is : ");
+		System.out.print(first + " " + next);
+		for (int i = 1; i <= n - 2; i++) {
```
Comment (ShrutiDJun20) [inline src/Fibonacci.java:12]: As per requirement, our output should start from 1 and end on 55 when user enters 10
your logic is correct, just not matching with the expected output.

File: src/Swap.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+public class Swap {
+	static void swapNum(int num1, int num2) {
+		num2 = num1 + num2;
+		num1 = num2 - num1;
+		num2 = num2 - num1;
+		System.out.println(" Numbers After Swapping : " + (num1) + "," + num2);
```
Comment (ShrutiDJun20) [inline src/Swap.java]: No need to put num1 in bracket here I think

File: src/Fibonacci.java (hunk 1 above)
Comment (PiyushPJun20) [reply src/Fibonacci.java:12]: Changes Incorporated.

File: src/Swap.java (hunk 2 above)
Comment (PiyushPJun20) [reply src/Swap.java]: Thanks for suggestion, Changes Incorporated.

File: src/Fibonacci.java (hunk 1 above)
Comment (PiyushPJun20) [reply src/Fibonacci.java:12]: image

**Review by ShrutiDJun20 — APPROVED** [review]

**Pull request merged**